compiler/gen-core/CMakeLists.txt

```diff
@@ -1,4 +1,9 @@
-find_package(HDF5 COMPONENTS CXX QUIET)
+# TODO This will be nnas_find_package
```

Comment:
(optional) `TODO This will be nnas_find_package` seems strange expression

What I would use: `TODO Revise this to use nnas_find_package`